en/docs/observe-and-manage/classic-observability-logs/configuring-log4j2-properties.md

```diff
@@ -384,10 +384,41 @@ See the instructions on [using wire logs to debug]({{base_path}}/develop/using-w
 
 Access logs related to service/API invocations are enabled by default in the Micro Integrator. Access logs for the PassThrough transport will record the request and the response on **two** separate log lines.
 
-By default, access logs are printed to the `http_acces_.log` file (stored in the `<MI_HOME>/repository/logs` folder). If required, you can use the log4j2 configurations to print the access logs to other destinations. Simply apply the following [logger](#log4j2-loggers) with an [appender](#log4j2-appenders).
+By default, access logs are printed to the `http_access.log` file (stored in the `<MI_HOME>/repository/logs` folder). If required, you can use the log4j2 configurations to print the access logs to other destinations. Simply apply the following [logger](#log4j2-loggers) with an [appender](#log4j2-appenders).
 
 -   <b>Logger Name</b>: PassThroughAccess
--   <b>Logger Class</b>: org.apache.synapse.transport.http.access
+-   <b>Logger Class</b>: org.apache.synapse.transport.http.access.logs
+
+Given below is the default log4j2 configuration for the access logs. 
+
+```
+logger.PassThroughAccess.name = org.apache.synapse.transport.http.access.logs
+logger.PassThroughAccess.level = DEBUG
+logger.PassThroughAccess.appenderRef.HTTP_ACCESS_LOGFILE.ref = HTTP_ACCESS_LOGFILE
+logger.PassThroughAccess.additivity = false
+
+# Appender config to HTTP access logs
+appender.HTTP_ACCESS_LOGFILE.type = RollingFile
+appender.HTTP_ACCESS_LOGFILE.name = HTTP_ACCESS_LOGFILE
+appender.HTTP_ACCESS_LOGFILE.fileName = ${sys:logfiles.home}/http_access.log
+appender.HTTP_ACCESS_LOGFILE.filePattern = ${sys:logfiles.home}/http_access_%d{yyyy-MM-dd}.log
+appender.HTTP_ACCESS_LOGFILE.layout.type = PatternLayout
+appender.HTTP_ACCESS_LOGFILE.layout.pattern = %msg%n
+appender.HTTP_ACCESS_LOGFILE.policies.type = Policies
+appender.HTTP_ACCESS_LOGFILE.policies.time.type = TimeBasedTriggeringPolicy
+appender.HTTP_ACCESS_LOGFILE.policies.time.interval = 1
+appender.HTTP_ACCESS_LOGFILE.policies.time.modulate = true
+appender.HTTP_ACCESS_LOGFILE.policies.size.type = SizeBasedTriggeringPolicy
+appender.HTTP_ACCESS_LOGFILE.policies.size.size=1MB
+appender.HTTP_ACCESS_LOGFILE.strategy.type = DefaultRolloverStrategy
+appender.HTTP_ACCESS_LOGFILE.strategy.max = 20
+appender.HTTP_ACCESS_LOGFILE.filter.threshold.type = ThresholdFilter
+appender.HTTP_ACCESS_LOGFILE.filter.threshold.level = DEBUG
+```
+
+To disable the access logs, set the log level of `logger.PassThroughAccess.level` to `OFF` or `WARN`.
+
+The output file name and the file pattern can be customized as required by changing the `fileName` and `filePattern` attributes of the `HTTP_ACCESS_LOGFILE` appender.
 
 ### Customizing the Access Log format
```

Comment:
Since we use log4j2 does changes made to `access-log.properties` are applicable still?.

Reply:
these two configurations are still applicable. access_log_file_date_format & access_log_pattern